pkg/download/download.go

```diff
@@ -23,11 +22,13 @@ func Decompressed(client *http.Client, u url.URL, path string) error {
 	if err := os.MkdirAll(filepath.Dir(path), 0o755); err != nil {
```

Comment:
I notice on line 20 (GitHub won't let me comment there), `gz` is included twice. I think the second one is supposed to be `xz`?

Reply:
good catch, will update comment

Reply:
fixed